technical-resources/protocol-overview/smart-contract-addresses-base.md

```diff
@@ -0,0 +1,105 @@
+# V2 Protocol
+
+## Administrative Contracts
+
+| Contract | Address | Commit Hash |
+| -------- | ------- | ----------- |
+| Globals (Proxy) | [`0x7f3C3636208A18c7941BF051807db56864061465`](https://basescan.org/address/0x7f3C3636208A18c7941BF051807db56864061465) | [`globals-v2 @ v2.0.0`](https://github.com/maple-labs/globals-v2/releases/tag/v2.0.0) |
+| MapleTreasury | [`0xa9466EaBd096449d650D5AEB0dD3dA6F52FD0B19`](https://basescan.org/address/0x3A5ab01B4E142F41f54a861c984D19C866Ec1736) | [`maple-core @ v1.0.0`](https://github.com/maple-labs/maple-core/releases/tag/v1.0.0) |
```

Comment:
the  link to `0x3A5ab01B4E142F41f54a861c984D19C866Ec1736` is correct but the listed address is not

Reply:
Why are we linking to `maple-core`? We have a Treasury repo and it seems like you are linking to them, e.g. https://github.com/maple-labs/treasury/releases/tag/v1.0.0

Reply:
I don't think we should link to `maple-core` ... in this case the deployed Treasury is a Gnosis Safe

Reply:
Yup Mike is correct this is one difference from the mainnet protocol, we just use a SAFE as the Treasury

Reply:
Should I take out the whole treasury section?